inductiva/api/methods.py

```diff
@@ -132,6 +132,18 @@ def kill_task(api_instance: TasksApi, task_id: str):
     logging.info("Task terminated.")
 
 
+def check_status(api_instance: TasksApi, task_id: str):
```

Comment:
Should we rename this to `check_task_status` for consistency with the naming used in the rest of the file (and in the API in general)?

Reply:
Or, actually, `get_task_status`, since we are getting the status and returning it?